pepper-apis/Dockerfile-build

```diff
@@ -0,0 +1,52 @@
+# To make our config generation "compatible" with official dsdetoolbox, extract what is needed from it (this is not the base image)
+FROM broadinstitute/dsde-toolbox:latest AS dsdetoolbox
+
+FROM maven:3.6.3-jdk-11 AS maven
+
+FROM google/cloud-sdk:alpine
+
```

Comment:
Tried to create as compact an image as possible as waiting for these images to spin up in CircleCi can be a big time waster.
The two big changes from the previous image were:
1) Using an Alpine base image
2) Leaving out the gcloud beta tools
Was able to bring the image down from over 800MB to below 300MB and on average much faster start up times in CircleCi (from, on average 90 seconds to on average 20 something seconds).
CIrcleCi image start times do tend to fluctuate a lot though, but this has made a big difference.